squad-list-metrics

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import logging
+import sys
+from squad_client.core.api import SquadApi
+from squad_client.core.models import Squad, ALL
+from squad_client.utils import getid
+
+SquadApi.configure(cache=3600, url="https://qa-reports.linaro.org/")
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def arg_parser():
+    parser = argparse.ArgumentParser(description="List all results for a squad build")
```

Comment:
s/results/metrics/ ?

When I see "results" I already imagine tests, but that may be me only.